CMakeLists.txt

```diff
@@ -458,6 +458,9 @@ if("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU")
     message(
       FATAL_ERROR "VELOX requires gcc > 8. Found ${CMAKE_CXX_COMPILER_VERSION}")
   endif()
+
+  # Find Threads library
+  find_package(Threads REQUIRED)
```

Comment:
Why? If ddb requires Threads it will/should look for it. Feel free to resolve if not for ddb but rather our changed code.

Reply:
I started seeing this in our code with this change. See https://app.circleci.com/pipelines/github/facebookincubator/velox/36404/workflows/3bdaa340-22b1-46df-8377-449100e5bbe9/jobs/241760

Reply:
Ah I see so it likely was a transitive dependency satisfied via DDB amalgamation that is now missing. Makes sense to add it in that case